src/app/assessment/assessment.component.ts

```diff
@@ -356,6 +356,17 @@ export class AssessmentComponent extends RouterEnter {
     continue?: boolean; // extra parameter to allow "options" appear as well-defined variable
     routeOnly?: boolean; // routeOnly: True, return route in string. False, return navigated route (promise<void>)
   } = {}): Promise<any> {
+    // skip "continue workflow" && instant redirect user, when:
+    // - review action (this.action == 'review')
+    // - fromPage = events (check AssessmentRoutingModule)
+    if (
+      this.action === 'review'
+      || this.doReview
+      || (this.action === 'assessment' && this.fromPage === 'events')
```

Comment:
We don't need `|| this.doReview` because if `this.doReview = true`, then `this.action` is 'review' for sure

Reply:
thanks @shawnm0705